### DISCUSSION TITLE
Projects: can not add external collaborator to my Github Organisation Project

### DISCUSSION THREAD

**Discussion by HiptJo** [community/community#36512] [Projects and Issues]
When following the instructions provided, the popup for adding a new user to a project, the user can not be added at the moment.
 
https://docs.github.com/en/organizations/managing-access-to-your-organizations-project-boards/adding-an-outside-collaborator-to-a-project-board-in-your-organization

When typing the username or email, no selection dropdown item appears.
When clicking on _Invite_, nothing happens at all.

Does anybody have similar issues?

**Comment by rivertwilight** [comment]
I just tried to add an external collaborator to a project of my own organization and all were successful.

Maybe you typed the target user's nickname instead of his/her **username**. Try to visit their homepage and check the URL to get his/her username.

![image](https://user-images.githubusercontent.com/52880665/196377820-f1468aba-8e87-420e-a460-9bf5af26983d.png)

![image](https://user-images.githubusercontent.com/52880665/196377999-305432e7-0035-4466-8967-6ec82bad1415.png)

**Reply by HiptJo** [reply]
That does not work for me, i don't know why tough.

When typing, no suggestions are displayed.
When debugging (viewing network requests) it seems like github does not suggest the account (see screenshot, empty array) - tried for 4 different accounts by now (one of them RiverTwilight just for testing purposes)

![image](https://user-images.githubusercontent.com/43265875/196416053-46ab4ea2-7fb0-4dc5-8327-030941e90167.png)
It is a private organisational project and it does not have any other members than myself at the moment...

**Reply by rivertwilight** [reply]
I checked your GitHub profile page but cannot find any organization you are in. 

![image](https://user-images.githubusercontent.com/52880665/196418822-ffdbb510-e8b2-45be-a274-e55c24e4db68.png)
 
Could you send the organization link?

**Reply by HiptJo** [reply]
https://github.com/hiptmairit

**Comment by ossanna16** [comment]
Hey there @HiptJo and thank you for asking a good question! I'm glad you found our community 🙂
